### PR TITLE
HexTimeTest

### PR DESCRIPTION
**Pull request by dixonl90**
My simple test showing the time with a background colour generated from the hours, minutes and seconds.

hextime

Twitter: @dixonl90

### CODE REVIEW

**Comment by blundell** [comment]
Could we get a gif? (instructions in readme)

File: mobile/src/androidTest/java/com/novoda/canvas/HexTimeTest.java [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+                    SimpleDateFormat sdf = new SimpleDateFormat("HH:mm:ss", Locale.UK);
+                    timeText.setText(sdf.format(mCalendar.getTime()));
+
+                    //Split out the time
+                    int hour = mCalendar.get(Calendar.HOUR_OF_DAY);
+                    int minute = mCalendar.get(Calendar.MINUTE);
+                    int second = 25 * mCalendar.get(Calendar.SECOND); //*25 to get more dramatic colour change
+
+                    //Set the old and new colours
+                    Integer currentColour = Color.rgb(hour, minute, second);
```
Comment (blundell) [inline mobile/src/androidTest/java/com/novoda/canvas/HexTimeTest.java:65]: ha cool idea

File: mobile/src/androidTest/java/com/novoda/canvas/HexTimeTest.java [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+import static android.view.ViewGroup.LayoutParams.MATCH_PARENT;
+
+/**
+ * HexTimeTest
+ *
+ * Show the current time with a background colour generated from the time.
+ *
+ * Created by Luke Dixon at DroidCon on 30/10/15.
+ */
+public class HexTimeTest extends NoiseActivityTest{
```
Comment (blundell) [inline mobile/src/androidTest/java/com/novoda/canvas/HexTimeTest.java:28]: don't think you use the `noise` functionality, should probably be `Novoda`

**Comment by dixonl90** [comment]
hextime

Here's a GIF! :)

**Comment by blundell** [comment]
:+1:

**Pull request merged**